.github/workflows/alpha-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Alpha Releases
+
+on:
+  schedule:
+    - cron: '0 20 * * 3' # weekly (Wednesday) 12 PM PST
+
+jobs:
+  test:
+    name: Test latest code
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check should run if HEAD is untagged
+        run: |
+          if [[ "$(git name-rev --tags --name-only $(git rev-parse HEAD))" != "undefined" ]]; then
+            exit 1
+          fi
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: 12.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies for master
+        run: yarn install --frozen-lockfile --non-interactive
+      - name: Basic Tests
+        env:
+          CI: true
+          ASSERT_ALL_DEPRECATIONS: true
+        run: yarn test
+
+  release:
+    name: Run publish script
+    runs-on: ubuntu-latest
+    needs: [ test ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Install dependencies for master
+        run: yarn install --frozen-lockfile --non-interactive
+      - name: Make sure git user is setup
+        run: |
+          git config --local user.email 'cron@example.com'
```

Comment:
reminder to update this. I believe there's a tomster one to use (see ember's config)

Reply:
updated to:

```
      - name: Make sure git user is setup
        run: |
          git config --local user.email 'tomster@emberjs.com'
          git config --local user.name 'Ember.js Alpha Releaser'
```